awx/main/analytics/collectors.py

```diff
@@ -4,6 +4,8 @@
 import os.path
 import platform
 import distro
+import pytz
```

Comment:
You should probably use the `utc` from django.utils.timezone instead of importing pytz.